src/io/utilities/functionSignatures.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Actually, it would be nice to have all `json` files in a common folder. @tpfau, what do you think?